mne/io/artemis123/artemis123.py

```diff
@@ -32,6 +35,12 @@ def read_raw_artemis123(input_fname, preload=False, verbose=None):
         on the hard drive (slower, requires less memory).
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
+    pos_fname : str or None (defulat None)
+        If not None, load digitized head points from this file
+    head_loc : bool (default False)
+        If True attempt to perform head localization using HPI coils. If no
+        HPI coils are in info['dig'] hpic coils are assumed to be in canonical
+        order of fiducial points (nas, rpa, lpa).
```

Comment:
> Is this the same as CTF?
I'm not sure what you mean. The concept is the same ie the assumption is that chpi coils are placed at the fiducial  locations but the order is most likely not that same.